soe/java/banking/README.md

```diff
@@ -156,3 +138,14 @@ What type of card would you like to add?
 Please enter your card number
     (Input card number with keypad)
 Okay I have added your card. Is there anything else I can help you with?
+
+## Additional Comments
+
+* If your Agent or Gateway calls are failing because they can't connect to the SOE then you're probably better off depploying the sample as a Cloud Foundry App. To do this we assume you have an IBM cloud account and you have installed on your machine IBM Cloud CLI. 
```

Comment:
Add a subsection ### Deploying to IBM Cloud

Reply:
You deploy this sample application to IBM Cloud as an easy way to connect to it from Voice Agent, as well as a Voice Gateway deployment.

Requirements:
+ Sign up for [IBM Cloud](https://cloud.ibm.com/registration).
+ Install the [IBM Cloud Standalone CLI](https://cloud.ibm.com/docs/cli/reference/ibmcloud/download_cli.html#install_use)